model/go.mod

```diff
@@ -0,0 +1,34 @@
+module github.com/jaegertracing/jaeger/model
+
+go 1.20
+
+replace github.com/jaegertracing/jaeger => ../
+
+require (
+	github.com/apache/thrift v0.19.0
+	github.com/gogo/protobuf v1.3.2
+	github.com/jaegertracing/jaeger v0.0.0-00010101000000-000000000000
```

Comment:
Does this mean we have circular dependency? Because in that case the whole exercise is pointless, the whole reason for separating model was so that OTEL contrib could import it with the rest of jaeger deps. I think model should have very minimal deps, we can probably separate them.

Reply:
i think we can remove this. this was automatically added when i ran `go mod tidy`.

Reply:
The only dependency of jaeger in `/model` is in tests like [here](https://github.com/jaegertracing/jaeger/blob/main/model/adjuster/package_test.go#L20). I think we need to have the testutils inside /model to remove the dependency.

Reply:
I see model/adjuster/package_test.go
Pointing to pkg/utils, which can explain the circular dependency

Reply:
to remove the circular dependency we will need to either copy the same things inside /model or make them separate packages as well